lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/python
 #
-# Copyright (c) 2016 Matt Davis, <mdavis@ansible.com>
```

Comment:
Why have you removed our copyright and replaced it with your own for making a few lines of change?